scipy/special/_add_newdocs.py

```diff
@@ -11091,7 +11091,31 @@ def add_newdoc(name, doc):
 
     See Also
     --------
-    nrdtrimn, ndtr
+    scipy.stats.norm : Normal distribution
+    ndtr : Standard normal cumulative probability distribution
+    ndtri : Inverse of standard normal CDF with respect to quantile
+    nrdtrisd : Inverse of normal distribution CDF with respect to
+               standard deviation
+
+    Examples
+    --------
+    `nrdtrimn` can be used to recover the mean of a normal distribution
+    if we know the CDF value `p` for a given quantile `x` and the
+    standard deviation `std`. First, we calculate
+    the normal distribution CDF for an exemplary parameter set.
+
+    >>> from scipy.stats import norm
+    >>> mean = 3.
+    >>> std = 2.
+    >>> x = 6.
+    >>> p = norm.cdf(x, loc=mean, scale=std)
+    >>> p
+    0.9331927987311419
+
+    Verify that `nrdtrimn` returns the original value for `mean`.
+    >>> from scipy.special import nrdtrimn
```

Comment:
this one and the other example requires newlines after text apparently 

https://output.circle-artifacts.com/output/job/ed62f044-399b-4392-9f29-061f67c3847a/artifacts/0/html/reference/generated/scipy.special.nrdtrimn.html#scipy.special.nrdtrimn